modules/misunderstood/src/views/lite/utils.tsx

```diff
@@ -0,0 +1,8 @@
+import snarkdown from 'snarkdown'
+
+export const renderUnsafeHTML = (message: string = ''): string => {
+  message = message.replace(/</g, '&lt;').replace(/>/g, '&gt;')
+
+  const html = snarkdown(message)
+  return html.replace(/<a href/gi, '<a target="_blank" href')
+}
```

Comment:
For another PR, 
I suggest that we extract this as a shared piece of code that can be used in our ui-shared-lite library and replace all markdown rendering by this.

Reply:
Do you want me to log this ?